### PR TITLE
Option to disable IP on userlist hover

### LINKED ISSUE #881: Option to turn off IP addresses in tooltip for admins
*Author's note: anything I've ~~crossed out~~ is not applicable to this issue.*

Please fill out the templates below to the best of your ability, based on whether your problem is with using the website or with running your own server.

## Website Problem ##

**Please confirm whether you've tried the following debugging steps:**

  - ~~[ ] Clearing cache and refreshing the page (On Firefox, press Ctrl+F5.  On Chrome, press F12, then right click the refresh button and click "Empty Cache and Hard Reload")~~
  - ~~[ ] Disabling all browser extensions~~
  - [x] Using a clean channel with no customizations
  
### Description of the Problem ###

  - What triggers the problem?
Hovering over a user to reveal the tooltip
  - What happens?
Shows the IP address of the user
  - What do you expect to happen instead?
Have an option to turn it off

### System Information ###

  - ~~**Operating System (Windows / Mac / Linux / Android / iOS):**~~
  - ~~**Web Browser (Firefox / Chrome / Other):**~~
  - ~~**Error Messages Displayed:**~~
  - ~~**Screenshot of JavaScript Console:**~~
  
_On Firefox, press `Ctrl+Shift+K` to open the JavaScript console.  On Chrome, press `Ctrl+Shift+J`._

### PR DESCRIPTION
**Pull request by lewisakura**
Resolves #881.

~~Adds an option under the `Chat` tab to toggle this option:~~
Adds an option under the `Moderator` tab to toggle this option:
image

### CODE REVIEW

**Comment by lewisakura** [comment]
Confirmed the feature works.

File: templates/useroptions.pug [hunk 1]
```diff
@@ -100,6 +100,10 @@ mixin us-chat
       +rcheckbox("us-sort-rank", "Sort userlist by rank")
       +rcheckbox("us-sort-afk", "Sort AFKers to bottom")
       .col-sm-4
+      .col-sm-8
+        p.text-info The following option only applies if you have permission to see IP addresses in the first place.
```
Comment (Xaekai) [inline templates/useroptions.pug]: This line shouldn't need to exist. There is a separate tab for options that are only applicable to moderators.

Comment (lewisakura) [reply templates/useroptions.pug]: Yeah that's a fair point. Let me move it over.

**Comment by lewisakura** [comment]
@Xaekai Just a question, where are default opts set? I'm just changing how my feature works slightly (on by default) but I can't seem to find where the default options are.

Edit: This always happens to me. I find it 10 seconds after I ask. Sigh.

**Pull request merged**